nilearn/plotting/img_plotting.py

```diff
@@ -188,12 +188,11 @@ def plot_img(img, cut_coords=None, output_file=None, display_mode='ortho',
         black_bg: boolean, optional
             If True, the background of the image is set to be black. If
             you wish to save figures with a black background, you
-            will need to pass "facecolor='k', edgecolor='k'" to pylab's
-            savefig.
+            will need to pass "facecolor='k', edgecolor='k'" to plt.savefig.
         colorbar: boolean, optional
             If True, display a colorbar on the right of the plots.
         kwargs: extra keyword arguments, optional
-            Extra keyword arguments passed to pylab.imshow
+            Extra keyword arguments passed to plt.imshow
```

Comment:
If you want to change this, I would rather go for matplotlib.pyplot.imshow, for things to be googleable by someone who is not versed in the shorthands of Python devs.